applications/StructuralMechanicsApplication/python_scripts/trilinos_structural_mechanics_solver.py

```diff
@@ -117,7 +117,7 @@ def _create_linear_strategy(self):
 
     def _create_newton_raphson_strategy(self):
         computing_model_part = self.GetComputingModelPart()
-        solution_scheme = self.get_solution_scheme()
+        solution_scheme = self.GetSolutionScheme()
```

Comment:
```suggestion
        solution_scheme = self._GetScheme()
```

the name of the C++ class is `Scheme`

furthermore as this is not in the public interface of the `PythonSolver` it should start with and underscore. This way it can still be called from outside without problems but shows the user it is an implementation detail

Reply:
Apart from the naming issue I very much agree with standardizing this!

Reply:
I'm fine with `GetScheme` (actually is what we used to have in the fluid). However, about the underscore, considering that it is to be called from outside in the ROM I made it "public". In any case I'm not against making it "protected".

Reply:
Done!

Reply:
It is python, hence you can also call protected methods
I think for the proposed application it is a good middle ground btw implementation detail and being able to call the method